src/components/common/ProfessorOverview/professorOverview.tsx

```diff
@@ -109,9 +109,7 @@ const ProfessorOverview = ({
         />
       )}
       <div className="flex flex-col items-center">
-        <p className="text-2xl font-bold self-center">
-          {searchQueryLabel(professor)}
-        </p>
+        
         {profData.state === 'loading' && (
```

Comment:
you'll need to add another skeleton in here to represent the loading professor name